site/layouts/shortcodes/orange-footer.html

```diff
@@ -0,0 +1,181 @@
+{{- /*
+  Usage: `orange-footer args`
```

Comment:
What do you think about renaming this file "footer.html" and if accepted replace any "orange-footer" text in the code by "footer" ?

Reply:
As discussed with Isabelle, there's already a file named footer.html (for Boosted footer). They are not in the same folder but IMO, it remains more clear in this way. However, we can try to have our footer in the shortcode and then rename the file footer.html.